modules/rosa-sts-cluster-terraform.adoc

```diff
@@ -13,7 +13,7 @@ endif::[]
 The following Terraform cluster example shows how to create your account-wide IAM roles and a ROSA cluster with a managed OIDC configuration.
 
 .Procedure
-. Optional: Because the Terraform files that are created in your current directory during this procedure, you can create a new directory to store these files.
+. Optional: Because the Terraform files are created in your current directory during this procedure, you can create a new directory to store these files.
```

Comment:
```suggestion
. Optional: Because the Terraform files get created in your current directory during this procedure, you can create a new directory to store these files and navigate into it by running the following command:
```

Reply:
Thanks @Srivaralakshmi!